spec/features/data_access/service_access_spec.rb

```diff
@@ -14,15 +14,22 @@
       wait_for_xhr
     end
 
-    context 'and submitting an ESI service request' do
+    context 'when choosing the ESI Service' do
       before :all do
         choose 'AE_Land.2 ESI Service'
         wait_for_xhr
+      end
+      it 'prepopulates the email address field' do
+        expect(find_field('Email Address').value).not_to be_blank
+      end
+      after :all do
```

Comment:
This block should be a `before` block under `context 'and submitting an ESI service request' do`